user-interface/src/main/java/life/qbic/datamanager/views/notifications/CloseableWithoutListeners.java

```diff
@@ -0,0 +1,16 @@
+package life.qbic.datamanager.views.notifications;
+
+/**
+ * Can close without triggering any close listeners.
```

Comment:
what is the benefit of this?

Reply:
Sometimes (for example after confirmation / successful operation). As calling `close` requires confirmation, I needed another method that provides instant closing without any user confirmation.